### PR TITLE
Link to external C++ example

### PR DESCRIPTION
**Pull request by emilk**
### What
* Closes https://github.com/rerun-io/rerun/issues/3758

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/4017) (if applicable)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4017)
- [Docs preview](https://rerun.io/preview/f89dc1b8d6f7d3e2669f138b8675da2ea8d188a5/docs) 
- [Examples preview](https://rerun.io/preview/f89dc1b8d6f7d3e2669f138b8675da2ea8d188a5/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: docs/content/getting-started/cpp.md [hunk 1]
```diff
@@ -3,14 +3,18 @@ title: C++ Quick Start
 order: 2
 ---
 
-## Installing Rerun
+## Setup
```
Comment (Wumpf) [inline docs/content/getting-started/cpp.md:6]: let's call this `Setup` over on rust.md as well! :)

File: rerun_cpp/README.md [hunk 2]
```diff
@@ -1,17 +1,10 @@
...
-This is not yet ready to be used.
+Read our [getting-started guide](https://www.rerun.io/docs/getting-started/cpp?speculative-link) for information on how to use the Rerun C++ SDK.
 
-## Requirements
+## Development
+### Requirements
 Run `scripts/setup.sh`.
 
-## Test it
+### Test it
```
Comment (teh-cmc) [inline rerun_cpp/README.md:9]: Shouldnt that say `just cpp-test` instead? We'll want to remove those shell scripts pretty soon...

File: docs/content/getting-started/cpp.md [hunk 3]
```diff
@@ -3,14 +3,18 @@ title: C++ Quick Start
...
 The Rerun C++ SDK depends on an install of the `arrow-cpp` library on your system using.
-If you are using [pixi], you can simply type `pixi global install arrow-cpp`.
+If you are using [Pixi](https://prefix.dev/docs/pixi/overview), you can simply type `pixi global install arrow-cpp`.
 Find more information about other package managers at the official Arrow Apache [install guide](https://arrow.apache.org/install/).
 
-If you're using CMake, you can add the following to your `CMakeLists.txt`:
+## Learning by example
+If you prefer to learn by example, check out our example repository which uses the Rerun C++ SDK to log some data from Eigen and OpenCV: <https://github.com/rerun-io/cpp-example-opencv-eigen>.
+
+## Using Rerun from CMake
```
Comment (teh-cmc) [inline docs/content/getting-started/cpp.md:16]:
```suggestion
## Using Rerun with CMake

**Review by teh-cmc — APPROVED** [review]

**Review by Wumpf — APPROVED** [review]
very nice!

**Pull request merged**